### PR TITLE
Include the booking location name on the customer emails

### PR DESCRIPTION
**Pull request by benbarnett**
So we can identify which booking location to contact in the case
that the customer responds to the email.

### CODE REVIEW

**Comment by benbarnett** [comment]
@benlovell for when you get a moment

File: app/views/booking_requests/customer.html.erb [hunk 1]
```diff
@@ -2,9 +2,7 @@
 0;font-size: 16px;line-height: 1.315789474;">Dear <%= @booking_request.name %>,</p>
 
 <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
-0;font-size: 16px;line-height: 1.315789474;">Thank you for requesting a
-guidance appointment with Pension Wise, we'll confirm the appointment slot
-within the next two working days.</p>
+0;font-size: 16px;line-height: 1.315789474;">Thank you for requesting a guidance appointment with Pension Wise. We’ll confirm your appointment in the next 2 working days.</p>
```
Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: Any particular reason this was reformatted?

Comment (benbarnett) [inline app/views/booking_requests/customer.html.erb]: Requested from @guidance-guarantee-programme/content-designers – shall I split out into a separate commit?

Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: No the content change is fine here, it was more about the line length being reformatted.

File: app/views/booking_requests/customer.html.erb [hunk 2]
```diff
@@ -19,6 +17,10 @@ within the next two working days.</p>
   class="emphasize" style="font-size: 24px;"><%= @booking_request.tertiary_slot
   %></strong></p>
 
+  <p style="color: #0B0C0C;font-family: Helvetica, Arial, sans-serif;margin: 15px
```
Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: The formatting is super weird here.

File: app/views/booking_requests/customer.html.erb (hunk 1 above)
Comment (benbarnett) [inline app/views/booking_requests/customer.html.erb]: Oh yeah, whoops!

File: app/views/booking_requests/customer.html.erb (hunk 2 above)
Comment (benlovell) [inline app/views/booking_requests/customer.html.erb]: Not gonna argue with that 😄

**Comment by benbarnett** [comment]
@benlovell could you take another look at this? Forgot it never went out :S
